README.md

```diff
@@ -73,6 +75,32 @@ https://www.overleaf.com/read/ygmygwtjbzfg
 
 https://www.overleaf.com/read/swqrxgqqvmyv
 
+The following instructions are all of the commands to install the dependencies used by this repository - this follows the same commands listed above:
+```
+conda update -y conda
+conda create -n expertise python=$PYTHON_VERSION -c conda-forge
+conda activate expertise
+mkdir ~/expertise-utils
+cd ~/expertise-utils
```

Comment:
I'm assuming that it's not necessary to create a folder in the home directory or is it?